state-chain/pallets/cf-cfe-event-emitter/src/benchmarking.rs

```diff
@@ -0,0 +1,21 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+use frame_benchmarking::benchmarks;
+
+benchmarks! {
+
+	clear_events {
```

Comment:
I think this will break when we generate new benchmarks: it needs to be called `remove_events_for_block` or alternatively, the method in `WeightInfo` needs to be renamed to `clear_events`. (I actually prefer the latter given that you removed the index per-block).

Reply:
Updated in https://github.com/chainflip-io/chainflip-backend/pull/4382/commits/0b710a255652616188f3f8d6658db4ff4f0c338c